frontend/entrypoint.sh

```diff
@@ -6,6 +6,9 @@ DEX_ISSUER_REPLACE=$(grep DEX_ISSUER .env | cut -d '=' -f 2)
 FRONTEND_REDIRECT_URI_REPLACE=$(grep FRONTEND_REDIRECT_URI .env | cut -d '=' -f 2)
 DEX_APPLICATION_ID_REPLACE=$(grep DEX_APPLICATION_ID .env | cut -d '=' -f 2)
 
+cp -r dist package.json .env /tmp/
+cd /tmp
+
```

Comment:
This is required due to [readOnlyRootFilesystem requirement](https://github.com/redhat-appstudio/infra-deployments/pull/3370), which is needed for https://github.com/redhat-appstudio/infra-deployments/pull/3370 to pass the kube-linter check.

I introduced [a new volume mount](https://github.com/redhat-appstudio/quality-dashboard/pull/312/files#diff-54d683ca0912dc7d95eeb59691820b0ee573002c53c30cf1b4f5fec35a062337R28-R30) for mounting `/tmp` where the frontend dist files are moved to and updating by the shell script.